src/style.css

```
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: bold;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+:root {
+  --big-text-padding-left: 62px;
+  --small-text-padding-left: 33px;
+}
+
+.input::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  color: hsl(212, 23%, 31%);
+}
+
+.form {
+  position: relative;
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.icon-search {
+  position: absolute;
+  background-image: url(./images/Search.svg);
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+  background-color: rgb(255, 255, 255);
+}
+
+.input {
+  background: hsl(0, 0%, 100%);
+  border: 1px solid hsl(210, 25%, 91%);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  width: calc(100% - var(--big-text-padding-left));
```

width should be 100%
```suggestion
  width: 100%;
```